gensim/models/keyedvectors.py

```diff
@@ -263,6 +255,121 @@ def add_word(word, weights):
         logger.info("loaded %s matrix from %s", result.syn0.shape, fname)
         return result
 
+    def similarity(self, word_1, word_2):
```

Comment:
The parameters are called `word_1, word_2` here, `w1, w1` in Euclidean an `term_1, term_2` in Poincare. A bit of consistency would be nice. The `distance()` is the same story.